### PR TITLE
Issue 230 imports

### PR DESCRIPTION
**Pull request by avmarchenko**
Issue 230 PR (2/3) correcting imports to route things to the internal package. Also updated the copyrights. Final PR will tackle test updates and removal of clone, dev install of exa from ci steps.

### CODE REVIEW

File: exatomic/algorithms/diffusion.py [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-# Copyright (c) 2015-2020, Exa Analytics Development Team
+# Copyright (c) 2015-2021, Exa Analytics Development Team
 # Distributed under the terms of the Apache License 2.0
 """
 Diffusion Coefficients
 ##########################
 Various algorithms for computing diffusion coefficients are coded here.
 """
-from exa.util.units import Length, Time
+from eaxtomic.exa.util.units import Length, Time
```
Comment (tjduigna) [inline exatomic/algorithms/diffusion.py:9]: sneaky typo

File: exatomic/exa/__init__.py [hunk 2]
```diff
@@ -14,8 +14,7 @@
 with open(os.path.join(os.path.dirname(__file__),
           'conf', 'logging.yml'), 'r') as f:
     _log = yaml.safe_load(f.read())
-_log['handlers']['file']['filename'] = os.path.join(tempfile.gettempdir(), 'exa.log')
+_log['handlers']['file']['filename'] = os.path.join(tempfile.gettempdir(), 'exatomic.exa.log')
```
Comment (tjduigna) [inline exatomic/exa/__init__.py:17]: logging needs some love anyway but probably can flatten to exatomic.log

**Review by tjduigna — APPROVED** [review]
Two suggestions:

1. Update the copyrights to 2015-2022, as we've already outlined a few months worth of work and PRs with 200+ files are painful :)
2. Hack the workflow with a sed replacement (or something) to get tables installed. I think that's how we got around pytables being named differently in conda and pip in travis.

Ongoing work to the github actions will change things considerably but would be nice to at least have the workflow running.

File: exatomic/exa/__init__.py (hunk 2 above)
Comment (avmarchenko) [reply exatomic/exa/__init__.py:17]: left as `exa.log` to avoid collision - will open issue to refactor.

**Comment by avmarchenko** [comment]
closes #230

**Pull request merged**